assets/scss/abstracts/_functions.scss

```diff
@@ -1 +1,30 @@
-/* _functions.scss */
+// -----------------------------------------------------------------------------
+// This file contains all application-wide Sass functions.
+// -----------------------------------------------------------------------------
+
+/// Native `url(..)` function wrapper
+/// @param {String} $base - base URL for the asset
+/// @param {String} $type - asset type folder (e.g. `fonts/`)
```

Comment:
I'd be worried about the slashes getting treated correctly with this function. I didn't see intellisense working on this.

Reply:
I didn't realize this was a thing: https://sass-lang.com/documentation/syntax/comments I'd just basically done a copy/paste from the boilerplate, but I think it makes sense to convert the `///` comments to `//` so they are highlighted correctly

Reply:
I didn't realize that either. https://sass-lang.com/documentation/syntax/comments#documentation-comments says the three slashes are for autobuilding documentation.

Reply:
I was actually referring to the `fonts/` being the parameter passed in. Using the argument `fonts` or `fonts/my-font` seems like a more logical argument.